Sources/SwiftCrossUI/Views/Modifiers/SheetModifier.swift

```diff
@@ -0,0 +1,170 @@
+extension View {
+    /// Presents a conditional modal overlay
+    /// onDismiss optional handler gets executed before
+    /// dismissing the sheet
+    public func sheet<SheetContent: View>(
+        isPresented: Binding<Bool>, onDismiss: (() -> Void)? = nil,
+        @ViewBuilder content: @escaping () -> SheetContent
+    ) -> some View {
+        SheetModifier(
+            isPresented: isPresented, body: TupleView1(self), onDismiss: onDismiss,
```

Comment:
Wrap each parameter onto its own line, and the closing `)` onto its own as well (only cause it's wrapped);

```
foo(
    param1,
    param2,
    ...
)
```